tests/Feature/DebitCardControllerTest.php

```diff
@@ -23,36 +25,88 @@ protected function setUp(): void
     public function testCustomerCanSeeAListOfDebitCards()
     {
         // get /debit-cards
+        $this->actingAs($this->user);
+        $response = $this->getJson(route('api.debit-cards.index'));
```

Comment:
for API tests using named routes isn't covering all the bases.

that is, you want your tests to break when you break BC in the API - any change in the url would be such case

Reply:
otherwise, if we change the path name and keep the logic intact. It seems that your case will take precedence because the client side cannot get the url from the route name